### ISSUE TITLE
What is the importance of Parallel SPPE?

### ISSUE THREAD

**Issue by zhenght17** [MVIG-SJTU/AlphaPose#110]
Hi! Thanks for sharing the code. After reading the paper, I've got a confuse and I hope you can answer it for me! Since the training code is not released yet, I can not fix the problem by reading the code myself.
  Now that Parallel SPPE shares the same STN with the original SPPE, and the layers of Parallel SPPE is freezed during the whole training phase, the purpose of this branch is to backpropagate the center-located pose error to STN. My question is, if we just need the loss between the label after the Parallel SPPE and the center-located ground truth label, why do we just skip the Parallel SPPE, and compare the output label of the STN and the center-located label? In my opinion, the weights of Parrael SPPE is freezed, which means it do not play an important part in the backpropagation part, so we can just throw it out.
  Looking forward to your reply!
  Best wishes!

**Comment by Fang-Haoshu** [comment]
Hi, you can check out the original training code at https://github.com/fang-haoshu/multi-human-pose
